frontend/src/pages/recommendations.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Reccomendations = () => {
```

Comment:
Both spellings in this file should be `Recommendations` not `Reccomendations` :)